### DISCUSSION TITLE
Amazing!

### DISCUSSION THREAD

**Discussion by amintikk** [skyline-emu/skyline#1262] [General]
It is incredible the great progress that this emulator is having, I would like to thank the programmers for the great effort they are making 👏👏

((Google Translate)) xd

**Comment by PixelyIon** [comment]
Thank you, the team is grateful to have a very supportive community! :smile:

**Reply by belmont31** [reply]
this is the best devolpper team ever ! i'm trying to test every single build with all games i have and when a game show somthing i'm like in heaven xD !!
thx you very much skyline team
